packages/wrangler/src/pages/functions/routes-transformation.ts

```diff
@@ -0,0 +1,8 @@
+/**
+ * TODO can we do better naming?
+ */
+export function convertRoutePathsToGlobPatterns(routes: string[]): string[] {
```

Comment:
We should probably input/output `UrlPath` here to be consistent with other routing code. Although these routes should never be derived from file paths, it doesn't hurt to be sure.